tests/sciml/test_sciml.py

```diff
@@ -202,8 +194,16 @@ def test_ude(test):
 
     # llh
     if test in (
+        # ?? cases where nn part of observable formula ??
```

Comment:
I think this might have been cases with neural network architecture we do not support

Reply:
Fair enough, they've all been implemented now in any case!